Cohort2/team4/server/src/app.ts

```diff
@@ -23,7 +22,6 @@ app.get<{}, MessageResponse>('/', (req, res) => {
   });
 });
 
-app.use('/api/v1', api);
 
 app.use(middlewares.notFound);
 app.use(middlewares.errorHandler);
```

Comment:
We'll need to add the trpc middleware here to have it accessible via express. You can use this after the other two middleware lines:

```
// created for each request
const createContext = ({
  req,
  res,
}: trpcExpress.CreateExpressContextOptions) => ({}); // no context
type Context = inferAsyncReturnType<typeof createContext>;

app.use(
  '/trpc',
  trpcExpress.createExpressMiddleware({
    router: appRouter,
    createContext,
  }),
);
```

You'll import the router from the 'trpc' file (see other comment).